ci-operator/step-registry/single-node/conf/aws/graceful-shutdown/single-node-conf-aws-graceful-shutdown-commands.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
@omertuc @osherdp why is it required? Didnt you solve it in another place already?